SU-SWS/stanford_profile
# D8CORE-1357: Adds validation to config form uri.

# READY FOR REVIEW

# Summary
- Adds validation for the redirect URL in the basic site settings.

# Review By (Date)
- Feb 26

# Urgency
- Med/Low

# Steps to Test

1. Check out this branch and clear caches
2. Navigate to `/admin/config/system/basic-site-settings`
3. Fill in a garbage url and submit
4. See validation error
5. Try a few combinations of stanford.edu domains 
6. See saved value

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- D8CORE-1357
- FYI @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
